contrib/cirrus/lib.sh

```diff
@@ -193,8 +193,8 @@ setup_gopath() {
         CRIO_SLUG $CRIO_SLUG
     "
     echo "Configuring persistent Go environment for all users"
-    sudo mkdir -p /var/tmp/go  # Works with atomic
-    sudo chown $USER:$USER /var/tmp/go
+    sudo mkdir -p /var/tmp/go/src  # Works with atomic
+    sudo chown -R $USER:$USER /var/tmp/go
```

Comment:
FYI- I doubt atomic will ever be used here...I think that's a copy-paste artifact I missed.